DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Features/MultipointFeatureToSpeckleConverter.cs

```diff
@@ -1,37 +1,26 @@
 using ArcGIS.Core.Geometry;
 using Speckle.Converters.Common.Objects;
 using Speckle.Core.Models;
-using Speckle.Converters.Common;
-using ArcGIS.Desktop.Mapping;
-using ArcMapPoint = ArcGIS.Core.Geometry.MapPoint;
 
 namespace Speckle.Converters.ArcGIS3.Features;
 
-[NameAndRankValue(nameof(Multipoint), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
-public class MultipointFeatureToSpeckleConverter : IHostObjectToSpeckleConversion, IRawConversion<Multipoint, Base>
+public class MultipointFeatureToSpeckleConverter : IRawConversion<Multipoint, List<Base>>
```

Comment:
Same here, we're returning a List<Base> but this should actually be a `List<SOG.Point>` to be super specific about what it returns.